### ISSUE TITLE
Install Crunchy Operator in CLAB and enable access for 2 project sets

### ISSUE THREAD

**Issue by mitovskaol** [BCDevOps/developer-experience#1786]
We want to assess the CrunchyDB product in our LAB cluster which requires installing the [Crunchy operator](https://github.com/CrunchyData/postgres-operator). NOTE: We will start evaluating the free version of the Crunchy operator first. ~If we can install the "enterprise" version alongside the free one, we will evaluate both in CLAB. If they cannot co-exist, the "enterprise" version will be deployed and evaluated later in KLAB once the assessment of RedHat Integration Suite in KLAB is complete.~

UPDATE: We could not negotiate a longer evaluation window with Crunchy Data for the "enterprise" version of the operator, so will start by evaluating the free version.

DoD:
- [x] Crunchy operator is installed in CLAB
- [x] 2 test project sets created in CLAB via the Registry (one with Karim as PO/TL and second is with Thor as PO/TL) (Olena will do it)
- [x] access to the Crunchy operator is enabled for these project sets

**Comment by StevenBarre** [comment]
Starting to document how to install monitoring https://github.com/bcgov-platform-services/platform-services-templates/pull/1

**Comment by StevenBarre** [comment]
Monitoring docs may need to move, or get some updates, but are a bit out of scope of this ticket now. The DoD is complete so I'll close this.
